source/_components/recorder.markdown

```diff
@@ -142,6 +142,7 @@ Purging does not necessarily remove all entries before a given date. For example
 | SQLite          | `sqlite:////PATH/TO/DB_NAME`                             |
 | MariaDB         | `mysql+pymysql://SERVER_IP/DB_NAME?charset=utf8`                 |
 | MariaDB         | `mysql+pymysql://user:password@SERVER_IP/DB_NAME?charset=utf8`   |
+| MariaDB         | `mysql://user:password@SERVER_IP:3307/DB_NAME?charset=utf8`   |
```

Comment:
Should mention the version number of MariaDB here, and remove the note under the table

Reply:
Thanks - some thoughts:

* If these lines are specifications rather than examples, then I guess the 3307 should be replaced by PORT.  The key thing is that the default port is different from MySQL, though.  So perhaps it's worth leaving the 3307 and changing the message under the table to point out that you may need to change this if you aren't using the default.

* The most important thing, though, is that the two mysql+pymysql MariaDB lines above *do not* work by default on typical current MariaDB installations (mine is v10), so I'd suggest they are removed or at least demoted to a lower place, and my guess is that, if the +pymysql variant works at all, it would also need :PORT to be added.

Many thanks!